src/lib/form-field/form-field-outline.scss

```diff
@@ -79,21 +79,18 @@ $mat-form-field-outline-subscript-padding:
   }
 
   .mat-form-field-outline-gap {
-    border-bottom: $mat-form-field-outline-width solid currentColor;
-
-    &::before {
-      content: '';
-      display: block;
-      width: 100%;
-      border-top: $mat-form-field-outline-width solid currentColor;
-      opacity: 1;
-      transition: opacity 300ms $swift-ease-out-timing-function;
-    }
+    // hack for Chrome's treatment of border width on non-integer displays
```

Comment:
Did you read about this somewhere, or just figured it out through experimentation? If you read about it, it would be nice to include a link for reference

Reply:
Nope, this was just through experimentation! Unfortunately I couldn't find any reported issue related to this. I don't imagine this scenario comes up too often.